src/main/java/com/demoqa/pages/HomePage.java

```diff
@@ -0,0 +1,43 @@
+package com.demoqa.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HomePage {
+    private static final String FORMS_CARD_TEXT = "//div[@class='card-body']/h5[text()='Forms']";
+    private static final String ELEMENTS_CARD_TEXT = "//div[@class='card-body']/h5[text()='Elements']";
+    private static final String WIDGETS_CARD_TEXT = "//div[@class='card-body']/h5[text()='Widgets']";
+    private static final String FRAME_CARD_TEXT = "//div[@class='card-body']/h5[text()='Alerts, Frame & Windows']";
+    private static final String SCROLL_INTO_VIEW_SCRIPT = "arguments[0].scrollIntoView(true)";
+    private final WebDriver driver;
```

Comment:
split constatns from fileds with an extra line
```suggestion

    private final WebDriver driver;
```